managed/ui/src/components/tables/CreateBackup/CreateBackup.js

```diff
@@ -282,7 +294,10 @@ export default class CreateBackup extends Component {
         <span>{props.data.label}</span>
       </components.SingleValue>
     );
-
+    const schedulingFrequencyUnitOptions = [
+      { value: 'Hours', label: 'Hours' },
```

Comment:
Use hours and minutes as constant. 
HOURS = 'Hours';
use this variable everywhere.

Reply:
you can also use some logic to generate this automatically.
Example,  
unites=[HOURS, MINUTE];
schedulingFrequencyUnitOptions = <for loop to generate value and label>

Reply:
Adding a new option can be done by simply adding a new entry in a constant list.